Src/Support/GoogleApis.Auth.DotNet4/OAuth2/AuthorizationDialog.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
```

Comment:
Please add the Google header, e.g. https://github.com/google/google-api-dotnet-client/blob/master/Src/Support/CommonAssemblyInfo.cs
Don't forget to change the year to 2016.